client/components/SingleUser.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+const SingleUser = () => {
+    return (
+        <div className="usercard">
```

Comment:
user-card or userCard? (referencing the comment about the camelcase vs using - like user-card)